### ISSUE TITLE
Skip to search is moving to the area below the search form

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1467]
Related to #1436. It seems that fix was implemented, but it's not moving to the correct place. It's just below the search form, which can be confusing to the user because it's the element they are moving to is hidden from view. Note that this is working correctly for Exhibits.

**Comment by thatbudakguy** [comment]
The weirdest thing about this is that, from what I can tell, it only happens when you're on the search page. On item view pages and the homepage, it works correctly.

It also does _focus_ the right element, but for some reason on the search page that scrolls it down somewhat. Maybe this has to do with the overall page length somehow?

**Comment by thatbudakguy** [comment]
From an HTML point of view, there is an anchor link constructed that points to the correct element...not sure what else can be done here.

**Comment by thatbudakguy** [comment]
It turns out that the newer CSS feature `scroll-margin` might help with this. Working on a fix in blacklight.

**Comment by thatbudakguy** [comment]
Fix is deployed.
